src/bench.js

```diff
@@ -0,0 +1,49 @@
+import benchmark from "benchmark";
+import { AwsCrc32 } from "@aws-crypto/crc32";
+import { AwsCrc32c } from "@aws-crypto/crc32c";
+import { Hash } from "@smithy/hash-node";
+
+const generateBuffer = (size) => {
+  const buf = Buffer.alloc(size);
+  for (let i = 0; i < size; i++) buf[i] = parseInt(Math.random() * 256);
+  return buf;
+};
+
+const suite = new benchmark.Suite();
+const testBuffer = generateBuffer(1024);
+
+const crc32 = new AwsCrc32();
+const crc32c = new AwsCrc32c();
+const sha1 = new (Hash.bind(null, "sha1"))();
+const sha256 = new (Hash.bind(null, "sha256"))();
+
+console.log(`Benchmark:`);
```

Comment:
you probably want to capture timing for bigger buffers. ideally at least 64k. 
ex. crt logs

https://github.com/awslabs/aws-checksums/blob/main/bin/benchmark/main.c#L94

Reply:
Updated in [`5b25b84` (#1)](https://github.com/trivikr/benchmark-checksums/pull/1/commits/5b25b84db51095db0bda66e8e431df90dfbf7161)